core/client/verify.go

```diff
@@ -37,21 +40,42 @@ var (
 // Verifier is a client helper library for verifying request and responses.
 type Verifier struct {
 	vrf         vrf.PublicKey
-	mapVerifier *client.MapVerifier
-	logVerifier client.LogVerifier
+	mapVerifier *tclient.MapVerifier
+	logVerifier tclient.LogVerifier
 }
 
 // NewVerifier creates a new instance of the client verifier.
 func NewVerifier(vrf vrf.PublicKey,
-	mapVerifier *client.MapVerifier,
-	logVerifier client.LogVerifier) *Verifier {
+	mapVerifier *tclient.MapVerifier,
+	logVerifier tclient.LogVerifier) *Verifier {
 	return &Verifier{
 		vrf:         vrf,
 		mapVerifier: mapVerifier,
 		logVerifier: logVerifier,
 	}
 }
 
+// NewVerifierFromConfig creates a new instance of the client verifier from a config.
```

Comment:
Being picky, "...from a Domain" would more accurately describe the func signature.
Are Domains and Configs interchangeable terms?
Given the overloaded uses of the word 'domain' is this a clue that they might be better named 'configs'?
Or is that a bit handwavy?

Reply:
Thanks, that's good feedback!